ui/e2e/services/details.spec.ts

```diff
@@ -0,0 +1,223 @@
+    .toBeTruthy();
+
+  const createdService = await findServiceWithApiRequest({
+    namespace,
+    match: (service) => service.filePath === "/redis-service.yaml",
+  });
+
+  if (!createdService) throw new Error("could not find service");
+
+  await page.goto(`/${namespace}/services/${createdService.id}`);
+
+  await expect(
+    page.getByRole("heading", { name: createdService.id, exact: true }),
+    "it renders the service id as a heading"
+  ).toBeVisible();
+
+  await expect(
+    page.getByRole("link", { name: "/redis-service.yaml", exact: true }),
+    "it renders a link to the service file"
+  ).toBeVisible();
+
+  await expect(
+    page.getByTestId("service-detail-header").getByText("imageredis"),
+    "it renders the service image name"
+  ).toBeVisible();
+
+  await expect(
+    page.getByTestId("service-detail-header").getByText("scale2"),
+    "it renders the service scale"
+  ).toBeVisible();
+
+  await expect(
+    page.getByTestId("service-detail-header").getByText("small"),
+    "it renders the service size"
+  ).toBeVisible();
+
+  await expect(
+    page.getByTestId("service-detail-header").getByText("redis-server"),
+    "it renders the service cmd"
+  ).toBeVisible();
+
+  await expect(
+    page
+      .getByTestId("service-detail-header")
+      .locator("a")
+      .filter({ hasText: "UpAndReady" }),
+    "it renders the UpAndReady status"
+  ).toBeVisible();
+
+  await expect(
+    page
+      .getByTestId("service-detail-header")
+      .locator("a")
+      .filter({ hasText: "1 environment variable" }),
+    "it renders one environment variable"
+  ).toBeVisible();
+
+  await expect(
+    page.getByText(`Logs for ${createdService.id}_1`),
+    "it renders the headline for the first pods logs"
+  ).toBeVisible();
+
+  await expect(
+    page.getByText("Ready to accept connections tcp"),
+    "it renders the log entries"
+  ).toBeVisible();
+
+  await expect(
+    page.getByText("received 8 log entries"),
+    "it renders the log summary"
+  ).toBeVisible();
+
+  await expect(
+    page.getByRole("tab", { name: "Pod 1 of 2" }),
+    "it renders the the first pod tab"
+  ).toBeVisible();
+
+  await expect(
+    page.getByRole("tab", { name: "Pod 2 of 2" }),
+    "it renders the the second pod tab"
+  ).toBeVisible();
+
+  await page.click("text=Pod 2 of 2");
+
+  /**
+   * since the logs from both pods are equal, we can only check, if changing
+   * the tab will trigger a network request to the logs of the second pod.
+   * We also check if the server responds with a 200 status code to make sure
+   * that the logs are comming in.
+   */
+  await expect(
+    await page.waitForResponse((response) => {
+      const logsApiCall = `/api/v2/namespaces/${namespace}/services/${createdService.id}/pods/${createdService.id}_2/logs`;
+      return response.status() === 200 && response.url().endsWith(logsApiCall);
+    }),
+    "after clicking on the second pod tab, a network request to the log was made"
+  ).toBeTruthy();
```

Comment:
I wanted to make sure the logs are actually requested, but we can also rely on the headline. It should be good enough. Removed it in 5af89caff2021448879eaa52b9fc96c0e155d9ca

If we move away from the docker driver, this test can be designed more accurate, because the logs should be different. Different timestamps, at least. This multiple pod feature is currently emulated in docker, and it is actually the same logs.